x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -17262,7 +17262,6 @@
     "xpack.monitoring.updateLicenseButtonLabel": "ライセンスを更新",
     "xpack.monitoring.updateLicenseTitle": "ライセンスの更新",
     "xpack.monitoring.useAvailableLicenseDescription": "すでに新しいライセンスがある場合は、今すぐアップロードしてください。",
-    "xpack.observability.alerts.breadcrumb": "アラート",
```

Comment:
@Bamieh most of these still exist but have been replaced by keys that more closely match Kibana i18n guidelines.